### PR TITLE
Fix travis

### PR DESCRIPTION
**Pull request by sayalilunkad**

### CODE REVIEW

**Comment by kbaikov** [comment]
Perhaps a comment on why the change was needed?

**Comment by sayalilunkad** [comment]
Travis currently is failing due to higher version of pbr being installed.
`pkg_resources.VersionConflict: (pbr 2.0.0 (/usr/local/lib/python2.7/dist-packages), Requirement.parse('pbr<2.0,>=1.0.0'))` Check any recently updated automation PR.

File: Makefile [hunk 1]
```diff
@@ -70,7 +70,7 @@ suseinstall:
 	sudo zypper install perl-JSON-XS perl-libxml-perl python-pip libvirt-python
 
 genericinstall:
-	sudo pip install bashate 'flake8<3.0.0' flake8-import-order jenkins-job-builder
+	sudo pip install 'pbr<2.0.0' bashate 'flake8<3.0.0' flake8-import-order jenkins-job-builder
```
Comment (aplanas) [inline Makefile]: maybe to be safe `pbr>=1.6,<2.0` is better here, but I am OK like that

Comment (sayalilunkad) [inline Makefile]: done, better to be safe

**Pull request merged**